client/src/tpu_connection.rs

```diff
@@ -17,5 +18,15 @@ pub trait TpuConnection {
 
     fn send_wire_transaction(&self, data: &[u8]) -> TransportResult<()>;
 
-    fn send_batch(&self, transactions: &[Transaction]) -> TransportResult<()>;
+    fn send_batch(&self, transactions: &[Transaction]) -> TransportResult<()> {
+        let buffers = transactions
+            .into_par_iter()
+            .map(|tx| bincode::serialize(&tx).expect("serialize Transaction in send_batch"))
+            .collect::<Vec<_>>();
+
+        let buffer_refs = buffers.iter().map(|buf| &buf[..]).collect::<Vec<_>>();
```

Comment:
I think if you leave the signature of send_batch unchanged, we do not even need this conversion to get the references:
pub async fn send_batch(&self, buffers: &[Vec])

Reply:
The problem is that send_wire_transaction_batch is also used by batch_send_single_target which does require &[&[u8]] because it's being used by forward_buffered_packets which get a vector of slices from the packet list.

Reply:
Fixed.